apache/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/php:7.3-apache
+FROM docker-remote.artifacts.developer.gov.bc.ca/php:7.3-apache
```

Comment:
would have preferred to update the build config FROM to use the artifactory tag instead of editing the docker file but no bigggieee